### DISCUSSION TITLE
Should the metamodel contains also the documentation of the single classes?

### DISCUSSION THREAD

**Discussion by ftomassetti** [Strumenta/StarLasu#4] [General]
Pros:
- We could use it in the documentation tool
- We could use it in the Playground

Cons:
- It would make the metamodel more complex
- It would make the metamodel heavier

**Comment by alessiostalla** [comment]
Are we referring to high-level documentation of each node class only, or detailed documentation for all the properties and methods?

**Reply by ftomassetti** [reply]
I was thinking of documentation for each Node class, but actually also documentation for properties would make sense, while methods are not included in the metamodel or the documentation
